app/src/main/convert.js

```diff
@@ -37,7 +38,21 @@ export default async function (exportOptions) {
     }
   });
 
-  app.kap.mainWindow.send('export-progress', {text: ''});
+  ipcMain.on('cancel-export', () => {
+    convertProcess.cancel();
+  });
+
+  await convertProcess.catch(err => {
```

Comment:
Better to just use try/catch here instead of `.catch()`. Then you don't need line 52 either.